addon/styles/_frost-button.scss

```diff
@@ -305,6 +297,54 @@ $frost-color-button-in-line-darken-2: darken($frost-color-blue-1, 15);
       box-shadow: none;
     }
   }
+  
+  // design = 'app-bar'
+  &.app-bar {
```

Comment:
Is this unique in any way other than the background color from a standard icon + text?  If not, then I'd prefer that this be included from the context where the design will be used - e.g. Design is for global common layouts and the one-offs are included from the unique context